app.py

```diff
@@ -311,24 +339,25 @@ def setWallet(self, wallet):
         Set the current wallet.
 
         Args:
-            wallet (Wallet): The wallet to use.
+            wallet (Wallet obj): The wallet to use.
```

Comment:
nah. `Wallet` is exactly what it is. It's also imported as `Wallet`. 

`from tinydecred.wallet import Wallet`